services/app-api/postgres/submissionWithRevisionsHelpers.ts

```diff
@@ -45,11 +45,11 @@ function convertToSubmission2Type(dbSub: StateSubmissionWithRevisions): Submissi
         stateCode: dbSub.stateCode,
         revisions: dbSub.revisions.map(r => { 
             let submitInfo: UpdateInfoType | undefined = undefined
-            if (r.submittedAt) {
+            if (r.submittedAt && r.submittedReason && r.submittedBy) {
```

Comment:
@macrael Isn't this where the default values should go that we discussed? We said for a new revision on creation that default `submittedReason` should be `Initial submission` and for `updatedBy` should be the current user

Reply:
I think we want to set those default values when we create the revision so that those default values are set in the database itself. This code would be right as is, converting the db fields to our domain submitInto type.